src/libsync/networkjobs.cpp

```diff
@@ -266,7 +279,9 @@ bool LsColXMLParser::parse(const QByteArray &xml, QHash<QString, ExtraFolderInfo
 
         if (type == QXmlStreamReader::StartElement && insidePropstat && insideProp) {
             // All those elements are properties
-            QString propertyContent = readContentsAsString(reader);
+            const QString propertyContent = (name == QLatin1String("checksums")) ? readContentsAsStringList(reader).join(" ")
```

Comment:
with 9ffa7bcd4d904ba927285e599b80e5d93d68f4d6 being merged, parsing of multiple checksum inside `checksums` is already working
please remove those changes

Reply:
@mgallien Wel... I don't know. Do you really think we should always prefer OwnCloud's way of solving issues? That `_checksums.indexOf('<', i);` on the checksum's side doesn't really look like the right place where we should parse XML tags. XML parsing logic seems to be a much better place for that. We don't have any XML tag symbols elsewhere for other key/value (s), right?

Reply:
yes but your change will not scale well if in future we would need such special case for other properties, we would end up with a lot of conditionals
this is why I remembered the change in the cherry-pick PR where there is no special handling only for one property type but a generic change that work well for any property type

Reply:
and by the way I forgot, but you are already doing better because we have automated tests thanks to your work
we are already better than them